src/common/com/intellij/plugins/haxe/lang/psi/impl/HaxeReferenceImpl.java

```diff
@@ -681,7 +694,7 @@ private void addChildClassVariants(Set<HaxeComponentName> variants, HaxeClass ha
     }
   }
 
-  private static void addUsingVariants(Set<HaxeComponentName> variants, @Nullable HaxeClass ourClass, List<HaxeClass> classes) {
+  private static void addUsingVariants(Set<HaxeComponentName> variants, Set<HaxeComponentName> variantsWithExtension, @Nullable HaxeClass ourClass, List<HaxeClass> classes) {
```

Comment:
What is the point of the new variable?  All of the data going to the first set is also going to the second.  Or is there supposed to be an 'if/else' statement surrounding lines 704 and 705?